site-src/geps/gep-718.md

```diff
@@ -0,0 +1,338 @@
+# GEP-718: Rework forwardTo segment in routes
+
+Issue URL: https://github.com/kubernetes-sigs/gateway-api/issues/718
+
+Status: implementable
```

Comment:
I meant to update this in the template, but haven't yet, do you mind making the following change to match the other GEPS?

```suggestion
Issue: [#718](https://github.com/kubernetes-sigs/gateway-api/issues/718)

Status: Implementable
```